backends/backends.c

```diff
@@ -719,6 +781,21 @@ void *backends_main(void *ptr) {
                 flags += MSG_NOSIGNAL;
     #endif
 
+                ssize_t written;
+#ifdef ENABLE_HTTPS
+                if(opentsdb_ssl.conn) {
+                    if(!opentsdb_ssl.flags) {
+                        written = SSL_write(opentsdb_ssl.conn,buffer_tostring(b), len);
+                    } else {
+                        written = send(sock, buffer_tostring(b), len, flags);
+                    }
+                } else {
+                    written = send(sock, buffer_tostring(b), len, flags);
+                }
+#else
+                written = send(sock, buffer_tostring(b), len, flags);
```

Comment:
1. You can't move it here. It should be after the `ENABLE_PROMETHEUS_REMOTE_WRITE` block.
2. Please get rid of `written = send(sock, buffer_tostring(b), len, flags);` 3 times in a row